Products/ZenUI3/browser/configure.zcml

```diff
@@ -398,7 +398,8 @@
 
 <browser:viewlet
     name="zenoss_data"
-    manager=".interfaces.IMainSnippetManager"
+    weight="3"
+    manager=".interfaces.IJavaScriptSrcManager"
```

Comment:
I also had to introduce `weight` attribute to make it render before `visualization-library`. To make this attribute work, they need to have the same managers.